lib/stencil-pull.utils.js

```diff
@@ -72,30 +76,88 @@ utils.downloadThemeConfig = (options, callback) => {
 
                                     let configFileData = '';
 
+                                    if (options.file && options.file.length) {
+
+                                        if (options.file !== entry.fileName) {
+                                            zipFile.readEntry();
+                                            return;
+                                        }
+                                        foundMatch = true;
+
+                                    } else if (options.exclude && options.exclude.length) {
+
+                                        /**
+                                         * Do not process any file or directory within the exclude option
+                                         */
+                                        for (let i = 0; i < options.exclude.length; i++) {
+                                            if ((new RegExp(`^${options.exclude[i]}`)).test(entry.fileName)) {
```

Comment:
@martinchuka regexp seems like a bad idea here

Reply:
> @martinchuka regexp seems like a bad idea here

@carsonreinke - I can't use `indexOf` since the directory inside the exclude can be way shorter than the file itself, I understand 'startsWith` and `endsWith` is far better than regex so I used those instead.